docs/src/terminology.md

```diff
@@ -271,15 +297,19 @@ The Nth [tick](terminology.md#tick) in the [ledger](terminology.md#ledger).
 
 ## token
 
-A scarce, fungible member of a set of tokens.
+A digitally transferable asset.
+
+## tower BFT
+
+The Solana consensus mechanism.  A proof-of-history optimized version of byzantine fault tolerance (PBFT) so named because it acts as a 'watch tower' over the Solana network, assuring validators that any attempt to cheat the proof-of-history clock will be punished.
```

Comment:
Where did you get this information from? I'm not sure this is true at all. Probably should remove the second sentence...

Reply:
I believe this term 'watchtower' came from the early writing of the founder or someone close to him and is the most succinct description of the Solana consensus mechanism i'm aware of.   See: https://u.today/solana-cracking-the-scalability-challenge-with-proof-of-history

I think it is helpful to have a clear definition of what the consensus mechanism is.   Agree?

Reply:
I'm pretty sure that was a spin by the article writer...
tower is because you're building a PoH tower, I believe...